plugins/woocommerce/tests/php/src/Internal/CLI/Migrator/Mocks/MockPlatformFetcher.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Mock Platform Fetcher class for testing.
+ *
+ * @package Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Mocks
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Tests\Internal\CLI\Migrator\Mocks;
+
+use Automattic\WooCommerce\Internal\CLI\Migrator\Interfaces\PlatformFetcherInterface;
+
+/**
+ * A mock fetcher class for testing purposes.
+ */
+class MockPlatformFetcher implements PlatformFetcherInterface {
```

Comment:
Can you elaborate on the usage of these mocks?

Reply:
Sorry it got mixed up with a test script I wrote, there are not being used. I've deteted them.